### PR TITLE
Make it clearer to users that we would welcome feedback on pain points & design quirks as well as "obvious" bugs

### PR DESCRIPTION
**Pull request by st--**

### CODE REVIEW

**Comment by st--** [comment]
Should perhaps make even clearer that "if you think you are doing it wrong" it's still worth flagging it up so that we can identify pain points that may be there for everyone & just go and fix them?

File: README.md [hunk 1]
```diff
@@ -77,13 +78,13 @@ There is an ["Intro to GPflow 2.0"](https://github.com/GPflow/GPflow/blob/develo
 
 ### Getting help
 
-**Bugs, feature requests, pain points:**
-Please use [GitHub issues](https://github.com/GPflow/GPflow/issues/) to flag up bugs/issues/pain points, suggest new features, and discuss anything else related to the use of GPflow that in some sense involves changing the GPflow code itself. Please make use of the labels such as `bug`, `discussion`, `feature`, etc.
+**Bugs, feature requests, pain points, annoying design quirks, etc:**
+Please use [GitHub issues](https://github.com/GPflow/GPflow/issues/) to flag up bugs/issues/pain points, suggest new features, and discuss anything else related to the use of GPflow that in some sense involves changing the GPflow code itself. Please make use of the labels such as `bug`, `discussion`, `feature`, `feedback`, etc.
```
Comment (joelberkeley-pio) [inline README.md]:
```suggestion
"We positively welcome comments or concerns about usability, and suggestions for changes at any level of design."
```

File: README.md [hunk 2]
```diff
@@ -13,10 +13,9 @@
...
- Alessandro Davide Ialongo, Alexander G. de G. Matthews, Alexis Boukouvalas, [Artem Artemev](http://github.com/awav/), Daniel Marthaler, David J
- . Harris, Eric Hambro, Hugh Salimbeni, Ivo Couckuyt, James Hensman, Keisuke Fujii, Mark van der Wilk, Mikhail Beck, Pablo Leon
- -Villagra, Rasmus Bonnevie, Sergio Pascual-Diaz, ST John, Tom Nickson, Valentine Svensson, Vincent Dutordoir, Zoubin
-  Ghahramani. GPflow is an open source project so if you feel you have some relevant skills and are interested in contributing then please do contact us.
+Alessandro Davide Ialongo, Alexander G. de G. Matthews, Alexis Boukouvalas, [Artem Artemev](http://github.com/awav/), Daniel Marthaler, David J. Harris, Eric Hambro, Hugh Salimbeni, Ivo Couckuyt, James Hensman, Keisuke Fujii, Mark van der Wilk, Mikhail Beck, Pablo Leon-Villagra, Rasmus Bonnevie, Sergio Pascual-Diaz, ST John, Tom Nickson, Valentine Svensson, Vincent Dutordoir, Zoubin Ghahramani.
+ 
+ *GPflow is an open source project, so if you feel you have some relevant skills and are interested in contributing then please do contact us (see ["The GPflow community" section](#the-gpflow-community) below).*
```
Comment (joelberkeley-pio) [inline README.md]:
```
GPflow is an open source project. If you have relevant skills and are interested ...
```

File: README.md [hunk 3]
```diff
@@ -66,6 +65,8 @@ GPflow leaves installing tensorflow to the user so that you can choose whether y
 
 ## Getting Started with GPflow 2.0
 
+*Note that GPflow 2.0 has not yet had an official release and interfaces, function signatures, variable names etc. may still change. Likewise, we are open to suggestions for how to improve the design of gpflow2!*
```
Comment (joelberkeley-pio) [inline README.md]: Likewise -> Indeed

Comment (joelberkeley-pio) [inline README.md]: "GPflow 2.0" and "gpflow2" both used - use one format?

File: README.md [hunk 4]
```diff
@@ -77,13 +78,14 @@ There is an ["Intro to GPflow 2.0"](https://github.com/GPflow/GPflow/blob/develo
 
 ### Getting help
 
-**Bugs, feature requests, pain points:**
-Please use [GitHub issues](https://github.com/GPflow/GPflow/issues/) to flag up bugs/issues/pain points, suggest new features, and discuss anything else related to the use of GPflow that in some sense involves changing the GPflow code itself. Please make use of the labels such as `bug`, `discussion`, `feature`, etc.
+**Bugs, feature requests, pain points, annoying design quirks, etc:**
+Please use [GitHub issues](https://github.com/GPflow/GPflow/issues/) to flag up bugs/issues/pain points, suggest new features, and discuss anything else related to the use of GPflow that in some sense involves changing the GPflow code itself. Please make use of the labels such as `bug`, `discussion`, `feature`, `feedback`, etc.
```
Comment (joelberkeley-pio) [inline README.md]: i suggest contributors apply tags, not the people raising the questions

File: README.md [hunk 5]
```diff
@@ -77,13 +78,14 @@ There is an ["Intro to GPflow 2.0"](https://github.com/GPflow/GPflow/blob/develo
...
+We positively welcome comments or concerns about usability, and suggestions for changes at any level of design.
 
 We aim to respond to issues promptly, but if you believe we may have forgotten about an issue, please feel free to add another comment to remind us.
 
-**How-to-use questions:**
-Please use [StackOverflow (gpflow tag)](https://stackoverflow.com/tags/gpflow) to ask questions that relate to "how to use GPflow", i.e. questions of understanding rather than issues that require changing GPflow code.
+**"How-to-use" questions:**
+Please use [StackOverflow (gpflow tag)](https://stackoverflow.com/tags/gpflow) to ask questions that relate to "how to use GPflow", i.e. questions of understanding rather than issues that require changing GPflow code. (You can always open an issue, but we may ask you to move your question to stackoverflow.)
```
Comment (joelberkeley-pio) [inline README.md]: StackOverflow -> Stack Overflow

stackoverflow -> Stack Overflow

File: README.md [hunk 6]
```diff
@@ -110,8 +112,7 @@ A few projects building on GPflow and demonstrating its usage are listed below.
 | [VBPP](https://github.com/st--/vbpp) | Implementation of "Variational Bayes for Point Processes".|
 | [DGPs_with_IWVI](https://github.com/hughsalimbeni/DGPs_with_IWVI) | Deep Gaussian Processes with Importance-Weighted Variational Inference|
 
-
-Let us know if you would like your project listed here.
+If you would like your project listed here, let us know - or simply open a Pull Request that adds your project to the table above!
```
Comment (joelberkeley-pio) [inline README.md]: Pull Request -> pull request

i think?

File: README.md (hunk 4 above)
Comment (st--) [reply README.md]: edited - happy with this?

File: README.md (hunk 3 above)
Comment (st--) [reply README.md]: We can unify this in another PR?

**Pull request merged**